### ISSUE TITLE
BEM события

### ISSUE THREAD

**Issue by ak-671** [bem/bem-forum-content-ru#278]
Добрый день.
Могу ли я делегировать определенное БЭМ событие всех блоков(без определения имени блока), которые находятся внутри блока родителя.

**Comment by tadatuta** [comment]
@ak-671 Нет, но можно примиксовать ко всем блокам служебный блок, который будет триггерить общее событие и подписываться на его.

**Comment by ak-671** [comment]
Спасибо, я рассматривал этот вариант, но он показался мне накладным(вместо одного объекта нужно создавать два), если блоков будет много (около 1000).
Возможно это ложное ощущение, тестов не проводил.

**Comment by zxqfox** [comment]
@tadatuta А через каналы нельзя?

**Comment by tadatuta** [comment]
@zxqfox можно, конечно
